justfile

```diff
@@ -151,7 +151,7 @@ build-client-prestate-asterisc-image kona_tag asterisc_tag out='./prestate-artif
 
 # Clones and checks out the monorepo at the commit present in `.monorepo`
 monorepo:
-  [ ! -d monorepo ] && git clone https://github.com/ethereum-optimism/monorepo
+  ([ ! -d monorepo ] && git clone https://github.com/ethereum-optimism/monorepo) || exit 0
```

Comment:
I think this change makes it so if the repo already exists, it exits... Should we not leave it so it checks out the commit?

Reply:
Right now it will exit still, but it won't be able to check out. If the `&&` fails, we `exit 1`. Just wanted there to be no visible error.